### PR TITLE
Portfolio: Javascript

### PR DESCRIPTION
**Pull request by muskan2532150**
#### Created a JavaScript file

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -390,25 +355,116 @@ hr {
...
+  line-height: 24px;
+  color: #6070ff;
+  background: #fff;
+  border: 1px solid #7f8cff;
+  border-radius: 8px;
+}
+
+/*  Navbar Hamburger Css ::clicked */
+.click {
+  background: rgba(96, 112, 255, 0.8);
```
Comment (tufoinnkuo10) [inline style.css]: - [ ] Great job so far. kindly use the same figma background color (#6070FF) so it follows the design

File: index.html [hunk 2]
```diff
@@ -15,23 +15,22 @@
...
-                <div class="nav-menu">
+              <button class="burger"><img src="images/Menu.png" alt="Hamburger"></button>  
+              <button><i class="fa-solid fa-xmark close-btn" ></i></button>  
                     <ul>
-                        <li><a href="#About">About</a></li>
-                        <li><a href="#Project">Project</a></li>
-                        <li><a href="#phone">Contact us</a></li>
+                        <li class="nav-link"><a href="#About">About</a></li>
+                        <li class="nav-link"><a href="#Project" >Project</a></li>
+                        <li class="nav-link"><a href="#phone">Contact</a></li>
```
Comment (tufoinnkuo10) [inline index.html:27]: - [ ] kindly add the font sizes to `32px` as in the figma version

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @muskan2532150 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ Menu close icon works perfectly well

Required Changes ♻️
Check the comments under the review
- [ required] JavaScript is included in this project.  Kindly update your linters as you are still using the simple HTML one. Please add JavaScript checks(ESLint) . You can see how to do that [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js). 

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css [hunk 3]
```diff
@@ -390,25 +355,117 @@ hr {
...
+
+/*  Navbar Hamburger Css ::clicked */
+.click {
+  background: #6070ff;
+  position: fixed;
+  backdrop-filter: blur(5px);
+  margin: 0;
+  padding: 45px;
+  transition: all 0.5s;
+}
```
Comment (vechicin) [inline style.css]: While you did implement the previous reviewer's requirement, you didn't do so properly. Please make sure you remove the `backdrop-filter` rule and add an `opacity` rule and set it to 0.9 to match the Figma design.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @muskan2532150 ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Good job implementing the previous reviewer's requirements.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — APPROVED** [review]
Hi @muskan2532150 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ Menu close icon works perfectly well
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**